docs/source/release.rst

```diff
@@ -2,152 +2,192 @@
-
+    $ make docs
 
-and for the pdf,
+To create the PDF documentation, do the following:
 
 .. code-block:: shell-session
 
-	$ make latexpdf
+    $ make -C latexpdf
 
-Since the ``docs/build`` folder is not kept track of, copy the pdf file
-with the documentation from ``docs/build/latex`` to the ``docs/pdf`` folder, saving it as ``mitiq.pdf``,
-thus replacing the previous version.
+Finally, Since the ``docs/build`` folder is not version controlled, copy the
+newly created PDF file from ``docs/build/latex`` to ``docs/pdf`` folder as
+``mitiq.pdf`` to overwrite the previous version.
 
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Create a distribution locally
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Create a PR with the above changes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Go to your local directory from the Terminal, e.g., ``github/mitiq/`` and there run
+After the required changes to ``VERSION.txt`` and ``CHANGELOG.md`` have been
+made, and the PDF documentation has been generated and moved to the correct
+location, it is recommended that the release author make a PR to master with
+these changes (rather than pushing directly to master) just in case. After
+this PR has been merged, the release author can go to the next step.
 
-.. code-block:: shell-session
+^^^^^^^^^^^^^^^^
+Create a new tag
+^^^^^^^^^^^^^^^^
 
-	$ python setup.py sdist bdist_wheel
+Tag the new commit to master (using ``git tag``) with a tag that matches the
+number ``VERSION.txt`` (with a preceding "v", so ``0.1.0`` is ``v0.1.0``) and
+push this tag to the Github repository.
 
-This will create a source distribution and a built distribution with wheel. This should create a ``build/`` and ``sdist/`` folder.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Create a source & built distribution locally
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Release the new version on Github
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-.. note::
-	You need to have write access to the ``mitiq``'s' Github repository to make a new release.
+From the top-level directory of the repository, run:
+
+.. code-block:: shell-session
 
-Make a new release on Github from the website, at https://github.com/unitaryfund/mitiq/releases.
+    $ python setup.py sdist bdist_wheel
 
-	- Choose a tag and add information on the release using the same style used for previous releases.
+This will create a "source" distribution and a "built" distribution using
+``wheel``. This should create a ``build/`` and ``sdist/`` folder.
 
-	- Make sure that a branch with the version number has been created and is referenced for the given release.
+**NOTE**: You will need to have installed ``wheel`` to create the "built"
+distribution.
 
-	- Github will create compressed files with the repository. Upload the ``mitiq.pdf`` file and add the locally generated distribution and the Python wheels.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Release the new version on Github
```

Comment:
Should this be automated with GH Actions? One example of some of this could be https://github.com/marketplace/actions/automatic-releases

Reply:
Yes it is possible, but I don't think it needs to be automated at this point -- it's only happening once a month. But, we could certainly open an issue to investigate it!